docs/contributing/development-environment.md

```diff
@@ -105,8 +122,9 @@ Once you have called `mx-tester up` you can run the integration tests.
 $ yarn test:integration
 ```
 
-After calling `mx-tester up`, if we want to play with mojlnir locally we can run the following and then point a matrix client to http://localhost:9999.
-You should then be able to join the management room at `#moderators:localhost:9999`.
+After calling `mx-tester up`, if we want to play with mojlnir locally we can run
```

Comment:
Mjolnir? We are Draupnir i think so is this rly correct?

Reply:
Well technically it's not mjolnir that's mentioned x3

Reply:
Kitty happy